src/configs/esbuild.es5.config.js

```diff
@@ -2,7 +2,7 @@
  * If not stated otherwise in this file or this component's LICENSE file the
  * following copyright and licenses apply:
  *
- * Copyright 2020 RDK Management
+ * Copyright 2020 Metrological
```

Comment:
2021

Reply:
@jitendraag in other files which were merged a couple of days ago, they are using Metrological 2020. Plus the copyright & blackduck tests has also passed.